code/game/objects/structures/cryopod.dm

```diff
@@ -0,0 +1,86 @@
+/obj/structure/cryopod_spawner/proc/justequip(var/mob/living/carbon/human/H, var/title, var/alt_title, var/outfit_type)
+	var/decl/hierarchy/outfit/outfit = outfit_type
+	. = outfit.equip(H, title, alt_title)
+
+/obj/structure/cryopod_spawner
+	icon = 'icons/obj/machines/excelsior/objects.dmi'
+	icon_state = "cryopod"
+	var/mob/spawnmob
+	var/spawnname
+	var/spawnsurname
+	var/outfit_type // The outfit the employee will be dressed in, if any
+	var/stat_modifiers = list(
+		STAT_MEC = 25,
+		STAT_COG = 40,
+		STAT_BIO = 25,
+	)
+
+/obj/structure/cryopod_spawner/proc/add_stats(var/mob/living/carbon/human/target)
+	if(!ishuman(target))
+		return FALSE
+	for(var/name in src.stat_modifiers)
+		target.stats.changeStat(name, stat_modifiers[name])
+
+	return TRUE
+
+
+/obj/structure/cryopod_spawner/attack_hand(mob/living/user as mob)
+	if(!spawnmob)
+		for(var/mob/observer/O in world)
+			if(!spawnmob)
+				//var/turf/T = get_turf(src.loc)
+				var/response = alert(O, "Are you -sure- you want to become a [spawnname] [spawnsurname]?.","Are you sure?","Near Ghost", "Random","Cancel")
+				if(response == "Cancel") return  //Hit the wrong key...again.
```

Comment:
Did you mean 'continue' here? Return means if one ghost declines rest don't even get asked.